plugins/pulp_rpm/plugins/distributors/yum/publish.py

```diff
@@ -625,6 +634,58 @@ def finalize(self):
                 add_metadata_file_metadata('updateinfo', self.context.metadata_file_path,
                                            self.context.checksum)
 
+    def _get_pkglist_to_publish(self, erratum_unit):
```

Comment:
I suspect it's often surprising to folks that the `erratum_unit` attribute `pkglist` isn't actually a list of packages, but is instead a list of lists of packages. We can help avoid propegating that confusion by renaming this to `_get_pkglists_to_publish`.